PostgresAssignment/src/services/class.ts

```diff
@@ -0,0 +1,22 @@
+import { query } from "../database/index";
+
+export async function getClasses(): Promise<Array<string>> {
+	const response = await query(`SELECT * FROM school.class order by name DESC`);
+	return response.rows;
+}
+
+export async function studentOfClass(classId: string): Promise<Array<string>> {
```

Comment:
return type is not always array. query failure would throw exception ???

Reply:
yes resolved it